cdap-ui/app/features/adapters/etlapps.less

```diff
@@ -125,25 +122,317 @@ body.theme-cdap.state-adapters-create {
 
   .modal {
 
+    .modal-dialog {
+      margin-top: 115px;
+    }
     .modal-lg {
       width: 95%;
     }
+    .modal-header {
+      background-color: @body-bg;
+      color: @cdap-header;
+      height: 60px;
+      padding: 0;
+      .border-radius(4px 4px 0 0);
+      div.pull-left {
+        height: 60px;
+        padding-left: 15px;
+        max-width: 725px;
+        * {
+          overflow: hidden;
+          text-overflow: ellipsis;
+          white-space: nowrap;
+        }
+      }
+      h3.modal-title {
+        font-weight: 500;
+        line-height: 1.3;
+        padding-top: 5px;
+      }
+      p {
+        color: @cdap-lightgray;
+        margin: 0;
+        max-width: 740px;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
+      .btn-group {
+        display: block;
+        text-align: right;
+        > div {
+          background-color: transparent;
+          border-left: 1px solid @cdap-lightgray;
+          text-align: center;
+          width: 60px;
+          .transition(all 0.2s ease);
+          &:first-child {
+            .btn {
+              border: 0;
+              .border-radius(0);
+            }
+            &:hover, &:focus {
+              .border-radius(0 4px 0 0);
+            }
+          }
+          &:hover, &:focus {
+            background-color: @cdap-header;
+            .btn {
+              color: white;
+            }
+          }
+        }
+      }
+      .btn {
+        background-color: transparent;
+        color: @cdap-header;
+        display: block;
+        font-size: 18px;
+        line-height: 60px;
+        padding: 0;
+        width: 100%;
+      }
+    }
+    .modal-body {
+      background-color: white;
+      overflow: hidden;
+      padding: 0 10px;
+      width: 100%;
+      .border-radius(0 0 4px 4px);
+      @shadow: 0 6px 20px 0 rgba(0,0,0,.2),0 5px 15px 0 rgba(0,0,0,0.19);
+      .box-shadow(@shadow);
+      @media (max-width: @screen-md-max) {
+        overflow: scroll;
+      }
+      @media (min-width: @screen-lg-min) {
+        height: 700px;
+        overflow: hidden;
+      }
+      > .row > [class*="col-"] {
+        padding-top: 20px;
+      }
+      .modal-content-scroll {
+        @media (min-width: @screen-lg-min) {
+          height: 700px;
+          overflow: scroll;
+        }
+      }
+      .col-lg-2 {
```

Comment:
I am not an avid fan of styling specific bootstrap grid classes for specific screen sizes. We style bootstrap columns in a more generic way like this,
```
[class*="col-lg"] {
....
}
```
And this applies to all bootstrap large classes across all screen sizes.  Even if a widget under the modal uses a bootstrap grid class it would be affected by it. In this case we are modifying col-lg-2 class alone and for specific screen sizes. If a widget in the modal happens to use the grid would it be affected by this in those screen sizes. Is it possible for to make it generic?